src/content/docs/en/guides/on-demand-rendering.mdx

```diff
@@ -211,7 +211,7 @@ You can also return a [Response](https://developer.mozilla.org/en-US/docs/Web/AP
 
 The example below returns a 404 on a dynamic page after looking up an ID in the database:
 
-```astro title="src/pages/[id].astro" {10-13}
+```astro title="src/pages/product/[id].astro" {10-13}
```

Comment:
I only updated because that's the path I had used before... it wasn't really a problem, just a matter of consistency.